### PR TITLE
Readme/rspec

### PR DESCRIPTION
**Pull request by jasonBirchall**
Improved README to allow friendlier introduction.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,38 @@
...
+## How to run smoke-tests
+The smoke tests will run inside a docker container against your current Kubernetes context (you can see your context by running `kubectl config get-contexts`).
+
+### Pre requisites
+- git
+- docker
+- `cluster-admin` access to a Cloud Platform cluster
+
+### Build or pull the image
+You'll need to have pulled the image locally, you can do this with the following:
```
Comment (digitalronin) [inline README.md:16]: The way this reads is a little bit confusing. I'd put the "pull" section immediately below this heading.

File: README.md [hunk 2]
```diff
@@ -2,6 +2,38 @@
...
+- run `make build`
+
+#### Pull
+- `docker pull ministryofjustice/cloud-platform-smoke-tests:1.0`
+
+### Run tests
+The smoke tests are separated into three sections.
+
+#### All tests
+All tests will run every rspec test in the `spec` directory.
```
Comment (digitalronin) [inline README.md]: I'd rephrase this as `make test` will run...

**Review by digitalronin — CHANGES_REQUESTED** [review]
This looks great. I've made a couple of small suggestions, for clarity.

File: README.md (hunk 1 above)
Comment (jasonBirchall) [reply README.md:16]: latest commit should've sorted this.

**Review by digitalronin — APPROVED** [review]
